packages/window-size-tool/rules.json

```diff
@@ -17,8 +17,7 @@
   {
     "width": "> 768",
     "terms": [
-      "//CLUSTER/@NUM_COLS = 2 and count(//CLUSTER[@NUM_COLS >= 2]//LIST) = 0 and count(//CLUSTER[@NUM_COLS > 2]) = 0 and count(//LIST[count(FIELD | CONTAINER) > 4]) = 0",
-      "count(//LIST[count(FIELD | CONTAINER) < 5]) > 0 and count(//LIST[count(FIELD | CONTAINER) > 4]) = 0"
+      "//CLUSTER/@NUM_COLS = 2 and count(//CLUSTER[@NUM_COLS >= 2]//LIST) = 0 and count(//CLUSTER[@NUM_COLS > 2]) = 0 and count(//LIST[count(FIELD | CONTAINER) > 4]) = 0"
```

Comment:
1. Generally combining this into a single term is good, for the same reason as we did #172 
2. But, it will now only apply if the modal contains a cluster with two columns but what if the modal doesn't contain a cluster at all but a list instead, then it won't be true? Maybe we need to change `//CLUSTER/@NUM_COLS = 2` to `(count(//CLUSTER) > 0 or count(//LIST) > 0)` to just check that we have either a cluster or a list (and thus likely don't have a VIM import)
3.

Reply:
Currently in CEF the output files from that first change look good, but yes that might happen in different stream/scenarios, so yes let me make the change and see the output again.